MIGRATION_GUIDE.md

```diff
@@ -6,20 +6,6 @@ across different versions.
 
 ## v0.92.0 ➞ v0.93.0
 
-### general changes
```

Comment:
I think that merge was incorrect for the migration guide, can you take a look?

Reply:
Yeah, now it should be under 93 -> 94